app/views/sign_up/registrations/new.html.erb

```diff
@@ -2,6 +2,21 @@
 
 <%= render 'shared/sp_alert', section: 'sign_up' %>
 
+<% if @a_b_test_bucket == :tabbed %>
+  <% if decorated_session.sp_name %>
+    <%= render 'sign_up/registrations/sp_registration_heading' %>
+  <% end %>
+
+  <%= render TabNavigationComponent.new(
+        label: t('account.login.tab_navigation'),
+        routes: [
+          { text: t('links.next'), path: root_path },
+          { text: t('links.create_account'), path: sign_up_email_path },
```

Comment:
To confirm: In the control, we embed the `request_id` as a parameter of the URL. Is that strictly necessary?

https://github.com/18F/identity-idp/blob/a5e83d3142d1fc79ca67c26c4df84f9b1a23f700/app/views/devise/sessions/new.html.erb#L88

Reply:
I would say yes, it's required?

embedding the `request_id` lets us link back to the SP after a user finishes signing in, including across devices/browsers (ex start on desktop but finish on mobile phone that has access to the email)

Reply:
But doesn't that get stuck in the session as soon as the user visits from the SP, so it wouldn't suddenly become lost if we don't carry it through as a request parameter?

1. Go to https://dev-identity-oidc-sinatra.app.cloud.gov/
2. Click Sign in
3. Go to https://idp.dev.identitysandbox.gov/sign_up/enter_email
4. Observe logo in header

Reply:
Hmmm yes good point, then maybe we could remove it from both as long as it's in the session still

Reply:
This is more complex than it appears at first glance, so I'm going to handle this separately in #8137 and probably add the `request_id` parameter here to allow this to move forward.